lib/help/index.js

```diff
@@ -4,6 +4,7 @@ import meme from './meme';
 
 export default {
   aide: (bot, msg, suffix) => {
+    var suffix = suffix.toLowerCase();
```

Comment:
`suffix` is already defined. You don't need to put `var`. This applies to the other two instances as well.